api/transaction.go

```diff
@@ -101,19 +101,127 @@ func (a *API) signTxHandler(w http.ResponseWriter, r *http.Request) {
 					}
 				}
 			}
+		case *models.Tx_NewProcess:
+			txNewProcess := tx.GetNewProcess()
+			// check the tx fields
+			if txNewProcess == nil || txNewProcess.Process == nil || txNewProcess.Nonce == 0 {
+				ErrInvalidTxFormat.With("missing fields").Write(w)
+				return
+			}
+			// check the tx subtype
+			switch txNewProcess.Txtype {
+			case models.TxType_NEW_PROCESS:
+				// generate a new faucet package if it's not present and include it in the tx
+				if txNewProcess.FaucetPackage == nil {
+					faucetPkg, err := a.account.FaucetPackage(organizationSigner.AddressString(), bootStrapFaucetAmount)
```

Comment:
This doesn't seem right, you need to calculate the cost for this transaction and create the exact faucet package amount. 

You need to import [this package ](https://github.com/vocdoni/vocdoni-node/blob/main/vochain/state/electionprice/electionprice.go) and initialize the ElectionPriceCalculator with the chain values (at this point, we probably need to define a vocdoni-node API endpoint to query for the chain parameters).